modules/get-started/pages/quick-start.adoc

```diff
@@ -226,75 +430,235 @@ include::get-started:partial$install-rpk-homebrew.adoc[]
+
+. Open MinIO at http://localhost:9001/browser to view your data stored in the S3-compatible object store.
++
+Login credentials:
++
+- Username:
++
+```
+minio
 ```
+- Password:
 +
-NOTE: The Redpanda broker returns a list of all broker addresses, so `rpk` can communicate with all brokers directly. Each broker returns its configured `advertise-*` address that matches the port to which the initial connection has been made.
+```
+redpandaTieredStorage7
+```
+
+See also: xref:manage:tiered-storage.adoc[]
+
+// (step {"action":"runShell", "command": "docker stop redpanda-2", "workingDirectory": "../docker-compose/three-brokers"})
+// (step {"action":"wait"})
+// (step {"action":"goTo", "url": "http://localhost:8080/overview"})
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "degraded-cluster.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+// (step {"action":"runShell", "command": "docker start redpanda-2", "workingDirectory": "../docker-compose/three-brokers"})
 
-== Explore your topic
+== Deploy a data streaming pipeline
 
-To explore your topic, you can use Redpanda Console, which you deployed in the `docker-compose.yml` file and is running locally on port 8080.
-Redpanda Console is a developer-friendly web UI for managing and debugging your Redpanda cluster and your applications.
+Deploying a data streaming pipeline in Redpanda involves configuring and running connectors that ingest, transform, and route data between various sources and sinks. Redpanda Connect simplifies this process by allowing you to manage these pipelines using declarative YAML configuration files and the `rpk` CLI.
 
-. http://127.0.0.1:8080/overview[Open Redpanda Console^] in a web browser.
+. Deploy the pipeline:
 +
-An overview of the cluster status, cluster health, and broker details is displayed.
+[,bash]
+----
+rpk connect run ../generate-profiles.yaml
+----
++
+NOTE: The pipeline configuration is in the `docker-compose/generate-profiles.yaml` file that you downloaded. This file is commented to explain how it works.
+
+. Consume 10 messages from the `profiles` topic to confirm that the pipeline is producing data to it.
++
+[,bash]
+----
+rpk topic consume profiles --num 10
+----
+
+== Extend your 30-day trial
+
+Your initial trial license is valid for 30 days. To extend it by another 30 days:
 
-. Go to http://127.0.0.1:8080/topics/chat-room[**Topics** > **chat-room**].
+NOTE: You can extend the trial license only once.
+
+. https://cloud.redpanda.com/try-enterprise[Sign up for a new trial license^].
+
+. Upload the new license to your running cluster using xref:reference:rpk/rpk-cluster/rpk-cluster-license-set.adoc[`rpk cluster license set`]:
 +
-The message that you produced to the topic is displayed along with some other details about the topic.
+[,bash]
+----
+rpk cluster license set <license>
+----
+
+This command applies the new license, extending your access to enterprise features for another 30 days.
```

Comment:
Before running this, the banner on my Overview page said, "Your Redpanda Enterprise trial will expire in 28 days." After successfully updating my license for another 30 days, the banner still said 28 days. Is that expected behavior?

Reply:
Oh for Redpanda Console to get the new license, you have to restart it. Thanks! I'll update with this info.